tox.ini

```diff
@@ -266,7 +262,7 @@ commands = {[packages]commands}
 skipsdist = True
 skip_install = True
 deps =
-    bandit==1.7.0
+    tomte[bandit]==0.1.1
```

Comment:
The benefit is now we have a single version of bandit that we can use throughout our entire stack, keeping everything nicely synced. Same for all other linters.